routes/auth.route.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+var express = require('express');
+var router = express.Router();
+var mongoose = require('mongoose');
+
+require('../models/users.models');
```

Comment:
I think this should go above all variable declarations. You want to require all your scripts first before declaring your variables. And it is a good practice to group them together.